source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -937,6 +946,15 @@ namespace
             else
             {
                 InputPtr input = destNodeGraph->addInput(nodegraphInput.getName(), nodegraphInput.getType());
+                RtTypedValue* uiFolderMetaData = nodegraphInput.getMetadata(RtToken(ValueElement::UI_FOLDER_ATTRIBUTE));
```

Comment:
Write out the nodegraph input uifolders.

Reply:
Seems like we should just write out all meta data.